src/main/java/dev/dfonline/codeclient/Commands.java

```diff
@@ -369,6 +369,48 @@ else if(field.getType().isEnum()) {
             return -1;
         })));
 
+        dispatcher.register(literal("search")
+                .then(argument("query",greedyString()).suggests((context,builder) -> suggestJump(JumpType.ANY, context, builder)).executes(context -> {
+                    if(CodeClient.location instanceof Dev dev) {
+                        var query = context.getArgument("query", String.class);
+                        var results = dev.scanForSigns(JumpType.ANY.pattern,Pattern.compile("^.*"+Pattern.quote(query)+".*$", Pattern.CASE_INSENSITIVE));
+
+                        if (results == null || results.isEmpty()) {
+                            Utility.sendMessage(Text.literal("No results."), ChatType.INFO);
```

Comment:
Please use translation keys, which are used in most files which aren't the config. You can add an entry to en_us.json and use them here.

Reply:
Implemented translation keys in the latest commit.